src/commands/profiles/port.js

```diff
@@ -0,0 +1,83 @@
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+const { cli } = require('cli-ux');
+
+class ProfilesPort extends BaseCommand {
+  async run() {
+    await super.run();
+
+    const query = await this.inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'confirmation',
+        message:
+          `This command will port profile(s) from keytar` +
+          ` to config file at location ${this.configFile.filePath}. Continue?`,
+        default: false,
+      },
+    ]);
+
+    if (!query.confirmation) {
+      this.logger.warn('Abort!');
+      return;
+    }
+
+    const { projects } = this.userConfig;
+
+    if (this.args.profile) {
+      const sanitizedProfileId = this.args.profile.trim();
+      const project = projects.find((p) => p.id === sanitizedProfileId);
+
+      if (!project) {
+        this.logger.error(
+          `Unable to port profile ${sanitizedProfileId} since it doesn't exist or has already been ported!`,
+        );
+        return;
+      }
+
+      await this.portProfile(project);
+    } else {
+      if (projects.length === 0) {
+        this.logger.info('All profiles already ported to config file!');
+        return;
+      }
+
+      for (const project of projects) {
+        await this.portProfile(project);
+      }
+
+      this.logger.info('Profiles port process complete!');
+    }
+  }
+
+  async portProfile(project) {
+    cli.action.start(`Porting ${project.id}`);
+
+    const creds = await this.secureStorage.getCredentials(project.id);
+    if (creds.apiKey === 'error') {
+      cli.action.stop('Failed: Unable to retrieve credentials from keytar.');
+      return;
+    }
+
+    // Also removes from `projects` along with adding to `profiles`
+    this.userConfig.addProfile(project.id, project.accountSid, project.region, creds.apiKey, creds.apiSecret);
+    await this.configFile.save(this.userConfig);
```

Comment:
Do we need to confirm the profile id is now available in profiles before moving ahead?

Reply:
Checked back again, I don't see any point of failure around saving at this point, and UTs for `addProfile` should assure this process for future PRs, since it already handles this responsibility for `projects` removal and `profiles` addition. 
In case of a bad-luck memory write for the userConfig, if should error out unexpectedly at that point itself before file write. Please let me know in case you see any place where a failure might occur before a write here!